src/Yoti.Auth/Profile/YotiProfile.cs

```diff
@@ -95,7 +101,8 @@ public YotiAttribute<string> EmailAddress
         }
 
         /// <summary>
-        /// DateOfBirth represents the user's date of birth as a DateTime. The time part of this DateTime will default to 00:00:00. This will be null if not provided by Yoti.
+        /// DateOfBirth represents the user's date of birth. Will be nil if not provided by Yoti. The
+        /// time part of this DateTime will default to 00:00:00. Will be null if not provided by Yoti.
```

Comment:
There's a mixture of nil/null in the documentation - is that correct?

Reply:
Oh yes, corrected in bf8af66